lib/chef/provider/package.rb

```diff
@@ -304,7 +304,7 @@ def expand_options(options)
         end
       end
 
-      # Check the current_version against either the candidate_version or the new_version
+      # Check the current_version against either the candidate_version or the new_version		 +
```

Comment:
that trailing `+` is weird.

Reply:
Whoops, I've removed that in my branch, will commit with the fixes :)